src/content/api/parser.md

```diff
@@ -532,7 +532,7 @@ Executed when parsing a `MemberExpression`.
 ```js
 const a = process.env;
 
-parser.hooks.new.for('process').tap('MyPlugin', expression => {});
+parser.hooks.expressionAnyMember.for('process').tap('MyPlugin', expression => {});
```

Comment:
Sorry for the delay. I've checked the [webpack repository](https://github.com/webpack/webpack/pull/9203/files#diff-313038f65941a3a6c139ebe61a484045ace246a429e31975f3606afdd95d977f), seems like `expressionAnyMember` was removed for a long time. So maybe we can remove it at the moment.